core/src/main/java/org/wildfly/extras/creaper/core/online/HttpModelControllerClient.java

```diff
@@ -43,15 +45,13 @@
 
     private final String url;
     private final RequestConfig requestConfig;
+    private final Registry<ConnectionSocketFactory> registry;
     private final CloseableHttpClient client;
 
-    HttpModelControllerClient(String host, int port, String username, String password) throws IOException {
-        this(host, port, username, password, NO_TIMEOUT);
-    }
-
-    HttpModelControllerClient(String host, int port, String username, String password, int timeoutMillis)
+    private HttpModelControllerClient(
+            String scheme, String host, int port, String username, String password, int timeoutMillis, SslOptions ssl)
```

Comment:
Come on, you should have learnt already that I have an OCD regarding code formatting :-)